.eslintrc.json

```diff
@@ -91,7 +92,7 @@
       },
       {
         "selector": "variable",
-        "format": ["camelCase", "UPPER_CASE"],
+        "format": ["camelCase", "UPPER_CASE", "PascalCase"],
```

Comment:
1. Can you revert the eslint changes and fix the file?
2. You seem to have introduced multiple code style rules for the entire repository based on one file.
3. Please fix the offending file

Reply:
@palisadoes I think that these changes are necessary because we need to make the first letter capital while creating a variable or component, and if we do so with the previous eslint rules we cannot create it because of the CamelCase Conditon, that's why previously the lines are commented in order to ignore the eslint conditons.